src/test/java/roomescape/repository/ReservationRepositoryTests.java

```diff
@@ -105,4 +105,41 @@ void deleteReservation() {
 		verify(this.jdbcTemplate).update(anyString(), eq(id));
 	}
 
+	@Test
+	void findReservations() {
+		// given
+		long memberId = 1L;
+		long themeId = 1L;
+		String dateFrom = "2024-06-01";
+		String dateTo = "2024-06-30";
+
+		ReservationTime reservationTime = ReservationTime.builder().id(1L).startAt("10:00").build();
+		Theme theme = Theme.builder().id(1L).name("테마1").description("첫번째테마").thumbnail("테마이미지").build();
+
+		Reservation reservation = Reservation.builder()
+				.id(1L)
+				.name("tester")
+				.date("2024-06-06")
+				.time(reservationTime)
+				.theme(theme)
+				.build();
+
+		List<Reservation> reservations = List.of(reservation);
+
+		given(this.jdbcTemplate.query(anyString(), any(RowMapper.class), eq(memberId), eq(themeId), eq(dateFrom), eq(dateTo)))
+				.willReturn(reservations);
+
+		// when
+		var findedReservations = this.reservationRepository.findReservations(memberId, themeId, dateFrom, dateTo);
```

Comment:
```suggestion
		var foundReservations = this.reservationRepository.findReservations(memberId, themeId, dateFrom, dateTo);
```

finded -> found 로 바꿔서 이름을 지어보는건 어떨까요?

Reply:
뭔가 어색하다고 생각했었는데...ㅋㅋ
전체적으로 수정했어요
https://github.com/next-step/spring-roomescape-auth/pull/37/commits/4a49e0954d42ecc7f07d30508eff6abdce78d849